<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>605</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>2DRacing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="StartPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="Background">
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="buddy">
           <cstring></cstring>
          </property>
         </widget>
        </item>
       </layout>
       <widget class="QLabel" name="SelectCar">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>40</y>
          <width>170</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select Your Car </string>
        </property>
       </widget>
       <widget class="QRadioButton" name="Car_1">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/car1.png</normaloff>:/car1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="Car_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/car2.png</normaloff>:/car2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
       <widget class="QRadioButton" name="Car_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>80</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/car3.png</normaloff>:/car3.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
       <widget class="QRadioButton" name="Car_4">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>80</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/car4.png</normaloff>:/car4.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>65</width>
          <height>62</height>
         </size>
        </property>
       </widget>
       <widget class="QCommandLinkButton" name="startRacing">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>380</y>
          <width>140</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Racing</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>200</y>
          <width>170</width>
          <height>30</height>
         </rect>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>210</y>
          <width>110</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Choose Difficulty : </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>230</y>
          <width>50</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Easy </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>230</y>
          <width>50</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Medium </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>230</y>
          <width>50</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Hard </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>300</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter Name : </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Myname">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>300</y>
          <width>140</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>You</string>
        </property>
        <property name="placeholderText">
         <string>ENTER YOUR NAME</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>38</x>
          <y>350</y>
          <width>110</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Search For Players</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>50</y>
          <width>130</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>High Score : 1729 (Hard)</string>
        </property>
       </widget>
       <zorder>Background</zorder>
       <zorder>label_6</zorder>
       <zorder>SelectCar</zorder>
       <zorder>Car_1</zorder>
       <zorder>Car_2</zorder>
       <zorder>Car_3</zorder>
       <zorder>Car_4</zorder>
       <zorder>startRacing</zorder>
       <zorder>horizontalSlider</zorder>
       <zorder>label</zorder>
       <zorder>label_2</zorder>
       <zorder>label_3</zorder>
       <zorder>label_4</zorder>
       <zorder>label_5</zorder>
       <zorder>Myname</zorder>
       <zorder>pushButton</zorder>
      </widget>
      <widget class="QWidget" name="RacerGame">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="graphicsView">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>30</y>
          <width>320</width>
          <height>70</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>35</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Game Over</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>160</y>
          <width>130</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Scoreboard :</string>
        </property>
       </widget>
       <widget class="QLabel" name="Player1score">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>210</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="Player1name">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>212</y>
          <width>150</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>You</string>
        </property>
       </widget>
       <widget class="QLabel" name="Player2name">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>252</y>
          <width>150</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Spawn1</string>
        </property>
       </widget>
       <widget class="QLabel" name="Player2score">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>250</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>380</y>
          <width>230</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Highscore : 1729 (Hard)</string>
        </property>
       </widget>
       <widget class="QLabel" name="Player3score">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>290</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="Player3name">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>292</y>
          <width>150</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Spawn2</string>
        </property>
       </widget>
       <widget class="QLabel" name="Player4score">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>328</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="Player4name">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>330</y>
          <width>150</width>
          <height>27</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Spawn3</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
